CONTRIBUTING.md

```
@@ -124,21 +146,17 @@ pdflatex -output-directory=./ -interaction=nonstopmode src/main.tex
 ### Требования
 - [TeX Live](https://www.tug.org/texlive/) (Рекомендуем) или [MiKTeX](https://miktex.org/)
 - Python 3.8+ (для генерации графиков)
-- Рекомендуемые пакеты: `pdflatex`, `bibtex`, `make`
+- Рекомендуемые пакеты: `pdflatex`, `biber`, `make`
```

markdownlint-cli2 (0.18.1)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)